code/game/gamemodes/endgame/xmas/victorianstation.dm

```diff
@@ -0,0 +1,112 @@
+/**
+	Victorianize your station
+		At one press of a button, turn every wall on the station to wood, all lighting to lanterns, all lamps to candles,
+		And the main hallways to snow.
+
+		Additionally, the bar becomes much more festive, with the addition of a christmas tree!
+
+		Also piss off your server host and set the entire server on fire.
+*/
+
+/datum/universal_state/auldlangsyne
+	name = "Older times"
+	desc = "The clock's ticking backwards!"
+
+	var/space_overlay1 = null
+	var/space_overlay2 = null
+
+
+/datum/universal_state/auldlangsyne/OnEnter()
+	space_overlay1 = image(icon='icons/turf/snowfx.dmi',icon_state="snowlayer1")
+	space_overlay2 = image(icon='icons/turf/snowfx.dmi',icon_state="snowlayer2")
+
+	var/target_zlevel = map.zMainStation
+
+	to_chat(target_zlevel, "<span class='sinister'>The air takes on an almost sepia tone...</span>")
+	//Snow up the halls
+	for(var/A in typesof(/area/hallway))
+		var/area/to_snow = locate(A)
+		if(to_snow)
+			for(var/turf/simulated/floor/F in to_snow)
+				F.ChangeTurf(/turf/unsimulated/floor/snow, tell_universe = 0)
```

Comment:
you're turning literally every simulated turf on the station into unsimulated
why would you do this
don't do this